### ISSUE TITLE
训练出现 rcnn box loss = 0

### ISSUE THREAD

**Issue by HZNUJeffreyRen** [guoruoqian/FPN_Pytorch#11]
why the rcnn box loss is always 0？
